pkg/didcomm/dispatcher/api.go

```diff
@@ -76,20 +76,20 @@ type OutboundCreator func(prov Provider) (Outbound, error)
 type DIDCommAction struct {
 	// DIDComm message
 	Message DIDCommMsg
-	// Callback function to be called by the consumer for further processing the message.
-	Callback Callback
+	// Continue function to be called by the consumer for further processing the message.
+	Continue Continue
+	// Fail invocation notifies the service that the consumer action event processing has failed or the consumer wants
+	// to stop the processing.
+	Fail Fail
```

Comment:
Suggest we rename this to an antonym of "continue" - suggest "Abandon" or "Discontinue" or "Stop"

Reply:
I had also mentioned "Stop" as a possibility.

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/42c5b3f270b75af9cd92cb0657abbcacffce467f..d4d6d2ee8a27820203156ad20b4f7a69e8277911#diff-6a4454074ff31ccb82bdbcc343255467R83) to Stop.